README.md

```diff
@@ -1,24 +1,7 @@
-# ForkDelta
-[![Chat on Discord](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://discord.gg/MPvAfMa)
-[![Join us on Reddit](https://img.shields.io/badge/reddit-ForkDelta-red.svg)](https://www.reddit.com/r/ForkDelta/)
+# Agorex-io
+[Chat on Discord](https://discord.gg/GVuj6UY)
+[Join us on Reddit](https://www.reddit.com/r/Agorex/)
```

Comment:
Again badge. Just change to:

[![Join us on Reddit](https://img.shields.io/badge/reddit-ForkDelta-red.svg)](https://www.reddit.com/r/Agorex/)